app/assets/stylesheets/custom.scss

```diff
@@ -56,13 +56,13 @@ body {
 
 // HEADER STUFF
 .dad-header {
-  height: 60vh;
+  height: 70vh;
   text-align: center;
 }
 
-.dad-header h1 {
-  color: #ffffff;
-  font-size: 5.2em;
+.dad-header h2, h1 {
+  color: #337ab7;
```

Comment:
Color literals like `#337ab7` should only be used in variable declarations; they should be referred to via variable everywhere else.